VideoLocker/res/layout/player_controller.xml

```diff
@@ -1,224 +1,224 @@
 
         <org.edx.mobile.view.custom.ETextView
             android:id="@+id/video_title"
             style="@style/video_title"
-            android:layout_width="match_parent"
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginRight="40dp"
             android:layout_marginEnd="40dp"
             android:layout_marginLeft="5dp"
+            android:layout_marginRight="40dp"
             android:layout_marginStart="5dp"
-            android:text="@string/app_name"
+            android:layout_weight="1"
+            tools:targetApi="17"
+            tools:text="This is a very long video title meant to be truncated from end" />
+
+        <org.edx.mobile.third_party.iconify.IconButton
+            android:id="@+id/lms_btn"
+            android:layout_width="20dp"
+            android:layout_height="20dp"
+            android:layout_marginEnd="12dp"
+            android:layout_marginRight="12dp"
+            android:background="@null"
+            android:contentDescription="@string/app_name"
+            app:iconColor="@color/edx_grayscale_neutral_white_t"
+            app:iconName="fa_share_square_o"
             tools:targetApi="17" />
 
-        <ImageButton
-                android:id="@+id/lms_btn"
-                android:layout_width="20dp"
-                android:layout_height="20dp"
-                android:layout_alignParentRight="true"
-                android:layout_alignParentEnd="true"
-                android:layout_centerVertical="true"
-                android:layout_marginRight="12dp"
-                android:layout_marginEnd="12dp"
-                android:background="@drawable/ic_lms_selector"
-                android:contentDescription="@string/app_name"
-                tools:targetApi="17" />
-
-        <ImageButton
+        <org.edx.mobile.third_party.iconify.IconButton
             android:id="@+id/share_btn"
             android:layout_width="20dp"
             android:layout_height="20dp"
-            android:layout_centerVertical="true"
-            android:layout_toLeftOf="@id/lms_btn"
-            android:layout_toStartOf="@id/lms_btn"
-            android:layout_marginRight="10dp"
             android:layout_marginEnd="10dp"
-            android:background="@drawable/ic_facebook_discuss_selector"
+            android:layout_marginRight="10dp"
+            android:background="@null"
             android:contentDescription="@string/app_name"
-            tools:targetApi="17"
-            />
-                
-    </RelativeLayout>
+            app:iconColor="@color/edx_grayscale_neutral_white_t"
+            app:iconName="fa_facebook_square"
+            tools:targetApi="17" />
+
+    </LinearLayout>
 
     <LinearLayout
         android:id="@+id/panel_bottom"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_alignParentBottom="true"
-        android:layoutDirection="ltr"

```

Comment:
The layout direction needs to be LTR as explained in [MA-837][].

[MA-837]: https://openedx.atlassian.net/browse/MA-837